nemo/collections/nlp/models/language_modeling/megatron_gpt_adapter_model.py

```diff
@@ -61,18 +62,30 @@ def __init__(self, cfg: DictConfig, trainer: Trainer):
                 )
 
         logging.info(f'Before adding adapters:\n{self.frozen_model.summarize()}')
+
+        if cfg.adapter_tuning.type == "parallel":
```

Comment:
For backward compatibility with our earlier adapters you can do cfg.adapter_tuning.get('type', default) here.

Reply:
Done!